### PR TITLE
[#2288] Support mousewheel scrolling in IE9

### PR DESCRIPTION
**Pull request by cadecairos**
Ticket on lighthouse: https://webmademovies.lighthouseapp.com/projects/65733/tickets/2288-ie9-scrollable-areas-dont-respond-to-mouse-wheel-scrolling

### CODE REVIEW

File: src/util/scrollbars.js [hunk 1]
```diff
@@ -79,11 +79,20 @@ define( [ "core/eventmanager" ], function( EventManager ) {
...
 
     outerElement.addEventListener( "mousewheel", function( e ){
-      if( e.wheelDeltaY ){
-        outerElement.scrollTop -= e.wheelDeltaY;
-        setHandlePosition();
-        e.preventDefault();
+      var delta = 0;
+
+      if ( e.wheelDelta ) {
+        delta = e.wheelDelta / 30;
```
Comment (mjschranz) [inline src/util/scrollbars.js]: My personal testing shows you could make this value something like 10 for a better scrolling experience. 30 makes it really slow when I'm scrolling a lot.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/97ba10527176f7d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/e8e744effb5fb71/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/97ba10527176f7d/output.txt

Total script time: 3.00 mins

Server is running on http://pr1058.butyr.org/templates/basic/
Run the Test Suite: http://pr1058.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/e8e744effb5fb71/output.txt

Total script time: 3.00 mins

File: src/util/scrollbars.js [hunk 2]
```diff
@@ -79,11 +80,11 @@ define( [ "core/eventmanager" ], function( EventManager ) {
...
 
     outerElement.addEventListener( "mousewheel", function( e ){
-      if( e.wheelDeltaY ){
-        outerElement.scrollTop -= e.wheelDeltaY;
-        setHandlePosition();
-        e.preventDefault();
-      }
+      var delta = e.wheelDeltaY || e.wheelDelta;
+
+      outerElement.scrollTop -= delta  / SCROLL_MODIFIER;
```
Comment (secretrobotron) [inline src/util/scrollbars.js]: Spacing after `delta`.

File: src/util/scrollbars.js [hunk 3]
```diff
@@ -5,7 +5,8 @@
 define( [ "core/eventmanager" ], function( EventManager ) {
 
   var VERTICAL_SIZE_REDUCTION_FACTOR = 3,
-      ACTIVE_CLASS = "butter-scollbar-active";
+      ACTIVE_CLASS = "butter-scollbar-active",
+      SCROLL_MODIFIER = 10;
```
Comment (secretrobotron) [inline src/util/scrollbars.js:9]: Should we use `SCROLL_MODIFIER` for the Firefox-specific scrolling as well?

Comment (cadecairos) [inline src/util/scrollbars.js:9]: No, the DOMMouseScroll event doesn't pass back values that need to be modified. In other words, smooth scrolling out of the box.

**Pull request merged**